Decred Wallet/Features/Settings/SettingsController.swift

```diff
@@ -82,25 +125,25 @@ class SettingsController: UITableViewController  {
         self.checkStartupSecurity()
 
         if Settings.networkMode == 0 {
-            network_mode_subtitle?.text = "Simplified Payment Verification"
+            network_mode_subtitle?.text = LocalizedStrings.spv
             self.certificate_cell.isUserInteractionEnabled = false
             self.server_cell.isUserInteractionEnabled = false
             self.connectPeer_cell.isUserInteractionEnabled = true
             self.server_ip.textColor = UIColor.lightGray
-            self.certificat_label.textColor = UIColor.lightGray
+            self.certificatDesc.textColor = UIColor.lightGray
```

Comment:
The only difference between `certificat` and `certificate` is an `e`. Why not just spell it completely? Also, please stop calling labels and headers description. A description is a long text meant to provide additional information on a concept, not the name of the concept itself.